packages/cli/templates/default-app/package.json.hbs

```diff
@@ -6,6 +6,7 @@
     "build": "lerna run build",
     "test": "cross-env CI=true lerna run test -- --coverage",
     "create-plugin": "backstage-cli create-plugin",
+    "clean": "backstage-cli clean",
```

Comment:
This is actually a template for the root package.json in a new app, so should be `lerna run clean` instead.

Reply:
@Rugvip I changed the script 😊